docker-compose.yaml

```diff
@@ -4,23 +4,17 @@ services:
 
   caddy:
     container_name: caddy
-    image: abiosoft/caddy:1.0.3-no-stats
-    ports:
-      - 80:80
-      - 443:443
+    image: caddy/caddy:2-alpine
```

Comment:
wrong docker repo. it's `caddy:2-alpine`

Reply:
Thank you!